include/strf/detail/facets/numpunct.hpp

```diff
@@ -337,7 +337,7 @@ class str_grouping: public strf::numpunct<Base>
 {
 public:
 
-    str_grouping(std::string grouping)
+	str_grouping(std::string grouping)
```

Comment:
Isn't a `STRF_HD` missing here?

Reply:
No, because we don't have `std::string`'s in device-side code. We could have a duplicate of `std::string`, but I'm not planning on doing anything like that for now.